Userland/Libraries/LibWebView/OutOfProcessWebView.cpp

```diff
@@ -459,6 +459,11 @@ void OutOfProcessWebView::notify_server_did_set_cookie(Badge<WebContentClient>,
         on_set_cookie(url, cookie, source);
 }
 
+void OutOfProcessWebView::notify_server_did_close_browsing_context(Badge<WebContentClient>)
+{
+    // TODO: close corresponding tab
```

Comment:
Would be awesome to keep feature parity between Ladybird and Browser, it should not be very complicated to add an `on_close` here as well and hook it up to close the tab. Feel free to split this into three commits when doing so (IPC preparation, Ladybird, Browser) :)